apps/system/js/notifications.js

```diff
@@ -460,6 +464,17 @@ var NotificationScreen = {
     messageContent.setAttribute('dir', 'auto');
     message.appendChild(messageContent);
     notificationNode.appendChild(message);
+    var countDiv = document.createElement('div');
+    countDiv.classList.add('detail-count');
+    countDiv.textContent = "1";
```

Comment:
Should we display a count when there is only one? Or should the numbers only appear when there are more than one?